src/plugins/vis_type_timelion/public/helpers/arg_value_suggestions.ts

```diff
@@ -73,18 +59,9 @@ export function getArgValueSuggestions() {
     es: {
       async index(partial: string) {
         const search = partial ? `${partial}*` : '*';
-        const resp = await savedObjectsClient.find<IndexPatternAttributes>({
-          type: 'index-pattern',
-          fields: ['title', 'type'],
-          search: `${search}`,
-          searchFields: ['title'],
-          perPage: 25,
-        });
-        return resp.savedObjects
-          .filter((savedObject) => !savedObject.get('type'))
-          .map((savedObject) => {
-            return { name: savedObject.attributes.title };
-          });
+        return (await indexPatterns.find(search, 25)).map(({ title }) => ({
```

Comment:
What's mean 25? Could you please add constant for this?

Reply:
sure